lib/json_rails_logger/version.rb

```diff
@@ -1,6 +1,8 @@
 # frozen_string_literal: true
 
 module JsonRailsLogger
+  NAME = "json_rails_logger"
```

Comment:
This duplicates information that is already in the Gemspec. You can read it like this:

```shell
ruby -e "puts Gem::Specification::load('json_rails_logger.gemspec').name"
```

Reply:
If you want, but you might as well hard code the name in the make file here. Since the name of the package is in the spec filename. So you no longer have a single source of truth for the package name and I was trying to avoid putting the package name in the makefile, as the inconsistent use of hyphen and underscore tripped me up at least once.

Reply:
I do indeed want. We have a single source of truth in the gemspec file. Duplicating information that is already present in the gemspec in other files, like `package.json` or `version.rb`, dilutes that without solving a problem.